src/Button/Button.module.scss

```diff
@@ -27,6 +27,14 @@ $color-night: #ccc;
   border-radius: $border-radius;
   cursor: pointer;
 
+  &.disabled,
+  &[disabled],
+  fieldset[disabled] & {
```

Comment:
`(L.31)` The button can be rendered `as` some specific element (e.g. `<a/>`), and from the HTML-standard point of view, applying `disabled` attribute to it will be incorrect.  I guess, we can handle `disabled` prop in a `Button` and map `.disabled` class to it via `cx(...)`.
`(L.32)` Are we gonna ever use a `fieldset` element 😄? I think it's better to be explicitly stated on a button that is disabled, then on some top-level parent component.  @Partysun, what do you think?